src/components/RibbonContainer.js

```diff
@@ -5,40 +5,62 @@ import g from 'glamorous';
 import { rhythm } from '../utils/typography';
 import { mediaQueries } from '../utils/styleHelpers';
 
-const InnerContainer = g.div(
-  {
-    /* Fallback */
-    margin: '0 auto',
-    padding: `${rhythm(2)} ${rhythm(3 / 4)}`,
-  },
-  {
-    [mediaQueries.small]: {
-      maxWidth: '560px',
-    },
-    [mediaQueries.medium]: {
-      maxWidth: '720px',
-    },
-    [mediaQueries.large]: {
-      maxWidth: '960px',
-    },
-    [mediaQueries.extraLarge]: {
-      maxWidth: '1120px',
-    },
-  }
+const InnerContainer = ({ children, margin, padding }) => (
+  <g.Div
```

Comment:
I prefer using `g.div` in this case since it takes care of rendering the children and passing the props along, but this works I suppose

Reply:
I tried that, but it wasn't working for me... might've been something else I had wrong, will check again

Reply:
I tried readdressing this but couldn't figure out how to use `g.div` and have it accept padding and margin as arguments